themes/luci-theme-bootstrap/htdocs/luci-static/bootstrap/cascade.css

```diff
@@ -624,9 +624,11 @@ select,
 	border-radius: 3px;
 	color: var(--text-color-high);
 	position: relative;
-	pointer-events: none;
+	pointer-events: auto; 
```

Comment:
`	pointer-events: auto; // needed for drag-and-drop in at least UIDynamicList`

Reply:
This only triggers when the "cbi-dynlist" class is applied. That should only be applied in a UIdynamicList function. So let not add that "at least" part

Reply:
Also I dont think // comments are allowed in CSS. /* */ only as far as I know